saritasa_drf_tools/filters/ordering.py

```diff
@@ -1,12 +1,85 @@
 import collections.abc
 import typing
+import warnings
 
+from django.conf import settings
 from django.core import exceptions
-from rest_framework import filters, views
+from django.db import models
+from rest_framework import filters, request, views
 
 
 class OrderingFilterBackend(filters.OrderingFilter):
-    """Custom OrderingFilter for better support of openapi."""
+    """Custom OrderingFilter with additional features.
+
+    Provides:
+    - Better support of openapi via drf-spectacular.
+    - Extra kwargs for ordering fields.
+
+    """
+
+    def get_ordering(
+        self,
+        request: request.Request,
+        queryset: models.QuerySet,
+        view: views.APIView,
+    ) -> collections.abc.Sequence[models.OrderBy]:
+        """Adjust ordering params.
+
+        Wrap ordering fields in `models.OrderBy` and pass extra ordering kwargs
+
+        """
+        ordering = (
+            super().get_ordering(
+                request,
+                queryset,
+                view,
+            )
+            or ()
+        )
+
+        is_null_first = getattr(
+            settings,
+            "SARITASA_DRF_ORDERING_IS_NULL_FIRST",
+            None,
+        )
+        is_null_last = getattr(
+            settings,
+            "SARITASA_DRF_ORDERING_IS_NULL_LAST",
+            None,
+        )
+
+        adjusted_ordering: list[models.OrderBy] = []
+        if unknown_fields := set(
+            getattr(self, "ordering_fields_extra_kwargs", {}).keys(),
+        ) - set(ordering):
```

Comment:
:fearful: I'm always scared by such long multiline statements. And this one is in `if` condition. Please simplify it